src/utils/tokenizers.py

```diff
@@ -53,8 +53,16 @@ def detokenize(self, tokens):
         '''
         return [self._detokenizer.unescape_xml(t) for t in tokens]
 
+class BertTokenizer(Tokenizer):
+    def __init__(self, bert_model_name):
+        from pytorch_pretrained_bert import BertTokenizer as HFBertTokenizer
+        do_lower_case = bert_model_name.endswith('uncased')
+        self._tokenizer = HFBertTokenizer.from_pretrained(bert_model_name, do_lower_case=do_lower_case)
+
+    def tokenize(self, sentence):
```

Comment:
We don't necessarily need a wrapper class here, we can put the constructor logic in a `get_tokenizer` function (see note above).

If we want to implement a `detokenize()` function, it's not the worst idea to send a PR to the `pytorch_pretrained_bert` maintainers.